.travis.yml

```diff
@@ -7,7 +7,9 @@ cache:
 install:
   - pip install pipenv
   - pipenv install
-script: lektor build
+script:
+  - lektor plugins add webpack-support
```

Comment:
You don't need to do this since `.lektorproject` (the configuration file) has `lektor-webpack-support` added.